src/unlimited.cpp

```diff
@@ -544,6 +544,46 @@ void UnlimitedSetup(void)
     // Start Internal CPU miner
     // Generate coins in the background
     GenerateBitcoins(GetBoolArg("-gen", DEFAULT_GENERATE), GetArg("-genproclimit", DEFAULT_GENERATE_THREADS), Params());
+
+
+    // Modify checkpoints depending on whether BCH or SV fork
+    if (Params().NetworkIDString() == "main")
+    {
+        if (nMiningSvForkTime == 0)
+        {
+            // Nov 15th 2018 activate LTOR, DSV op_code
+            ModifiableParams().ModifiableCheckpoints().mapCheckpoints[556767] =
```

Comment:
Nit: can we make a local reference for ModifiableParams().ModifiableCheckpoints() so it is not repeated so much (makes the source look nicer)

Reply:
ok done